interface/backend/api/serializers.py

```diff
@@ -54,3 +54,11 @@ class Meta:
         read_only_fields = ('created',)
 
     centroid = ImageLocationSerializer()
+
+    def create(self, validated_data):
+        case_data = validated_data.pop('case')
```

Comment:
Why `pop` the data? I mean, versus just indexing it with `validated_data['case']` (ie. please add a comment to the code itself if there is a reason; this is non-intuitive)

Reply:
:man_facepalming: Yep, `pop`ing definitely makes no sense here :)